tasks/container/Containerfile

```diff
@@ -50,12 +50,14 @@ RUN dnf -y update && \
         python3-pytest-asyncio \
         python3-pytest-timeout \
         python3-pyyaml \
+        python3-types-pyyaml.noarch \
```

Comment:
The .noarch suffix is weird here, let's please drop that.

Reply:
Copy-paste error.